Proxy batch

Hoooao opened the pull request:
To be tested

dqian3 reviewed (approved):
Everything looks good here, let's try this out soon

dqian3 committed:
Proxy Batching

fix loopback for nng

add yaml field and msg type for batch, no logic change

proxy logic impl

replica logic impl

befor etest

Uncompress logs

tmp

batch tested, no perf improv for rpc

Hoooao committed:
fix

Hoooao committed:
resilience config update

Hoooao committed:
update quorum config

Hoooao committed:
update
